instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/MapboxHistoryTest.kt

```diff
@@ -244,6 +244,8 @@ class MapboxHistoryTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.j
 
     private companion object {
         private const val CUSTOM_EVENT_TYPE = "custom_event_type"
-        private const val CUSTOM_EVENT_PROPERTIES = "custom_event_properties"
+        private const val CUSTOM_EVENT_PROPERTIES = """
+            {"name":"John", "age":30, "car":null}
+        """
```

Comment:
Should this be added?

```suggestion
        """.trimIndent()
```

Reply:
I get a compilation error
> Const 'val' initializer should be a constant value

Reply:
yeah best thing would be to verify the json values (instrumentation test here so we can use a json library).

because this will fail if nav-native decides to remove the white space from the json strings. 

nit-picking here though, can consider it out of scope